contracts-periphery/test/UniswapWithdrawHook.fork.1.t.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "test/utils/DSTestPlus.sol";
+import "test/utils/DeployUmbraTest.sol";
+import "src/UniswapWithdrawHook.sol";
+
+contract UniswapWithdrawHookTest is DeployUmbraTest {
+  using SafeERC20 for IERC20;
+  UniswapWithdrawHook withdrawHook;
+
+  IUmbra umbraContract;
+  ISwapRouter swapRouter;
+  IQuoter quoter;
+  IERC20 dai;
+
+  uint256 toll;
+
+  // Mainnet Addresses
+  address public constant DAI = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+  address public constant WETH9 = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+  address public constant Router = 0x68b3465833fb72A70ecDF485E0e4C7bD8665Fc45;
+  address public constant Quoter = 0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6;
+  uint24 poolFee = 3000;
+
+  function setUp() public override {
+    super.setUp();
+    umbraContract = IUmbra(address(umbra));
+    swapRouter = ISwapRouter(Router);
+    quoter = IQuoter(Quoter);
+    withdrawHook = new UniswapWithdrawHook(ISwapRouter(swapRouter));
+    dai = IERC20(DAI);
+    // Owner approves tokens
+    withdrawHook.approveToken(dai);
+    deal(address(DAI), address(this), 10e23);
+  }
+
+  function testFuzz_HookTest(
+    uint256 amount,
+    uint256 swapAmount,
+    uint256 feeBips
+  ) public {
+    address feeRecipient;
+    address recipient;
+    uint256 NumOfAddrs = 10;
```

Comment:
nit on naming convention, same on L48 where we use this var
```suggestion
    uint256 numOfAddrs = 10;
```

Reply:
Thanks for the reviews. Fixed it here https://github.com/ScopeLift/umbra-protocol/pull/341/commits/753879207c7ce0d74824acc41ed421338a73eefc